code/part-two/client/source/services/requests.js

```diff
@@ -51,6 +51,38 @@ const dropAddress = entity => {
     }, {});
 };
 
+/**
+ * Takes an encoded Batch and submits it to the REST API, optionally waiting
+ * until the batch is committed to resolve (waits by default).
+ *
+ * Resolves to `true` if submission was successful, otherwise throws an error.
+ */
+export const submitBatch = (encodedBatch, shouldWait = true) => {
+  return axios({
+    method: 'POST',
+    url: 'api/batches',
```

Comment:
needs to have `/` in `/api/batches` to prevent requests being made to `localhost:8000/moji/api/batches` and `localhost:8000/collection/api/batches`